__tests__/helpers.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const MemoryFileSystem = require("memory-fs");
+const webpack = require("webpack");
+
+exports.PluginEnvironment = class PluginEnvironment {
```

Comment:
Yes, and that's probably better anyway. Im thinking that's probably something for a new PR, though - since it will involve probably slightly more tooling to be created for that.